hoomd/md/alchemy/methods.py

```diff
@@ -0,0 +1,107 @@
+# Copyright (c) 2009-2022 The Regents of the University of Michigan.
+# Part of HOOMD-blue, released under the BSD 3-Clause License.
+
+"""Alchemical MD integration methods."""
+
+import hoomd
+from hoomd.md.alchemy.pair import AlchemicalDOF
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.data import syncedlist
+from hoomd.md.methods import Method
+from hoomd.variant import Variant
+
+
+class Alchemostat(Method):
+    """Alchemostat base class.
+
+    Note:
+        :py:class:`Alchemostat` is the base class for all alchemical integration
+        methods. Users should use the subclasses and not instantiate
+        `Alchemostat` directly.
+
+    """
+
+    def __init__(self, alchemical_particles):
+        self._alchemical_particles = syncedlist.SyncedList(
```

Comment:
When did we switch back to calling this `alchemical_particles`? `alchemical_dof` is more descriptive.

Reply:
I switched it back. If we have a strong reason to keep calling it `alchemical_particles`, them I'd be OK with it. Otherwise, mark as resolved.